app/models/post.rb

```diff
@@ -1,16 +1,11 @@
 class Post < ApplicationRecord
-  belongs_to :author, class_name: 'User', foreign_key: :author_id
+  after_save :increment_posts_counter
```

Comment:
- [ ] I was unable to create a post because of this callback. You defined an `after_save` callback for `increment_posts_counter`, but that method does not exist. Kindly add the method.

  ![rails-seed-tchilo](https://user-images.githubusercontent.com/36542986/172626790-eef7176c-b001-472c-9548-f76c4c2eda8f.jpg)

  ## SUGGESTION
  Example method
  ```
  def increment_posts_counter
    author.increment!(:post_counter)
  end
  ```